text-templates/attach-aws-batch.md

```diff
@@ -0,0 +1,8 @@
+You can run the following command in your local build directory to re-attach to
+this AWS Batch job later to see output and download results:
+
+```
+nextstrain build --aws-batch --attach ${AWS_BATCH_JOB_ID} .
+```
+
+You can also view this job in the [AWS Batch console](https://console.aws.amazon.com/batch/home?region=us-east-1#jobs/detail/${AWS_BATCH_JOB_ID}).
```

Comment:
The region `us-east-1` got masked in the summary of the [test seasonal-flu run](https://github.com/nextstrain/seasonal-flu/actions/runs/5138333906) because it is the value of the `AWS_DEFAULT_REGION` secret of the repo. I guess GitHub automatically masks any value that matches a secret value. 

It doesn't need to be a secret, so we can switch to the new GitHub Action [variables](https://docs.github.com/en/actions/learn-github-actions/variables) if we really need it as part of the workflows.

Reply:
Actually, the `AWS_DEFAULT_REGION` is not used anywhere in the seasonal-flu repo, so I'm just going to remove it. However, still good to know about this secret masking.